### PR TITLE
Update to unidecode 1.2.0 and use its features

### PR DESCRIPTION
**Pull request by ale-rt**
This branch resolves the merge conflicts of the one already pushed #21.

Jenkins job PRs:
https://github.com/plone/buildout.coredev/pull/401
https://github.com/plone/plone.i18n/pull/31
```
https://github.com/plone/buildout.coredev/pull/401
https://github.com/plone/plone.i18n/pull/31
```

Closes #21

The tests passed, see: https://github.com/plone/buildout.coredev/pull/573

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by ale-rt** [comment]
Test running here https://github.com/plone/buildout.coredev/pull/573

File: news/31.bugfix [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Make it work with newwest Unidecode 1.0.22. Use its method instaed of duplicating it in here.
```
Comment (mauritsvanrees) [inline news/31.bugfix:1]: Typo: `instaed` -> `instead`.
And I would not call it a bugfix, but a feature. But it is debatable.

**Review by mauritsvanrees — APPROVED** [review]
Amazing! Finally this incompatibility with newer unidecode gets solved. I am not aware of using it (my Dutch language is close enough to ascii that it does not matter much), but I am aware of the version annotation for unidecode in the coredev buildout that has been there for years. Great that we can finally get rid of it.

Thanks a lot @gogobd @ale-rt @jensens !

One minor comment inline about one of the news snippet. Could also be fixed after merge.

File: news/31.bugfix (hunk 1 above)
Comment (jensens) [reply news/31.bugfix]: renamed and corrected

**Pull request merged**